packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -345,32 +348,39 @@ export function makeRumPublicApi(
          * @param options.startTime epoch timestamp of the start of the custom vital
          * @param options.duration duration of the custom vital
          * @param options.context custom context attached to the vital
-         * @param options.details  Details of the vital.
+         * @param options.description  Descritpion of the vital
          */
         ;(rumPublicApi as any).addDurationVital = monitor(
-          (name: string, options: { startTime: number; duration: number; context?: object; details?: string }) => {
+          (name: string, options: { startTime: number; duration: number; context?: object; description?: string }) => {
             addTelemetryUsage({ feature: 'add-duration-vital' })
             strategy.addDurationVital({
               name: sanitize(name)!,
               type: VitalType.DURATION,
               startClocks: timeStampToClocks(options.startTime as TimeStamp),
               duration: options.duration as Duration,
               context: sanitize(options && options.context) as Context,
-              details: sanitize(options && options.details) as string | undefined,
+              description: sanitize(options && options.description) as string | undefined,
             })
           }
         )
 
         /**
-         * @deprecated
          * Stop a custom duration vital
          * stored in @vital.custom.<name>
          *
-         * @param name name of the custom vital
+         * @param nameOrRef name of the custom vital or the reference to the custom vital
          * @param options.context custom context attached to the vital
-         * @param options.stopTime epoch timestamp of the stop of the custom vital (if not set, will use current time)
+         * @param options.description Description of the vital
          */
-        ;(rumPublicApi as any).stopDurationVital = noop
+        ;(rumPublicApi as any).stopDurationVital = monitor(
+          (nameOrRef: string | DurationVitalReference, options?: { context?: object; description?: string }) => {
+            addTelemetryUsage({ feature: 'start-duration-vital' })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ typo?
```suggestion
            addTelemetryUsage({ feature: 'stop-duration-vital' })
```